examples/trivial/main.py

```diff
@@ -46,6 +47,35 @@ def add_spans():
 
 
 
+def add_spans_withLightStep():
```

Comment:
if this isn't called anywhere, we should just remove it... if you want to test the code below, just fold it into `add_spans` (that was the spirit of my comment from a few days ago)